kalitectl.py

```diff
@@ -471,7 +471,15 @@ def start(debug=False, daemonize=True, args=[], skip_job_scheduler=False, port=N
         # Write the new PID
         with open(PID_FILE, 'w') as f:
             f.write("%d\n%d" % (os.getpid(), port))
-    
+
+    # Start the job scheduler (not Celery yet...)
+    if not skip_job_scheduler:
+        manage(
+            'cronserver_blocking',
+            args=[],
+            as_thread=True
+        )
+
```

Comment:
Was moved because of some vague idea that starting this thread should happen after become_daemon.. not sure if it really changes anything.